### PR TITLE
DCOS-38114 : fix(tables/styles): fix FF collapsed table td 

### PR DESCRIPTION
**Pull request by weblancaster**
**Issue:**
The label is collapsed on Firefox, breaking the ui.

**Fix:**
Removes the `max-width: 0;` which was causing the ui to be `width 0`.

**Result:**
FF and Chrome tested.

**Test:**
Please test on Chrome and FF to make sure is wor

**Before**
screen shot 2018-06-11 at 3 02 21 pm

**After**
screen shot 2018-06-12 at 10 56 32 am

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

Closes DCOS-38114

### CODE REVIEW

**Comment by nLight** [comment]
It's in review but in the future sprint 🤔

**Comment by DanielMSchmidt** [comment]
@nLight I moved it into the sprint to reflect the work being done on this.

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ build/
 npm-debug.log
 node_modules/
 src/js/config/Config.dev.js
+src/js/config/Config.dev.ts
```
Comment (DanielMSchmidt) [inline .gitignore]: Why do we need this? Can't we just remove the file when we switch branches?

Comment (nLight) [reply .gitignore]: Could be annoying when switching between 1.11 and master in the same dir. I'd say we merge it and bump the release until the deadline. As it is not a blocker is it?

Comment (weblancaster) [reply .gitignore]: Correct, not a blocker

**Pull request merged**